src/OpenLoco/Audio/VehicleChannel.cpp

```diff
@@ -1,18 +1,84 @@
 #include "VehicleChannel.h"
 #include "../Entities/EntityManager.h"
 #include "../Interop/Interop.hpp"
+#include "../Map/TileManager.h"
+#include "../Ui/WindowManager.h"
 #include "../Vehicles/Vehicle.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Audio
 {
-    static std::pair<SoundId, Channel::Attributes> sub_48A590(const Vehicles::Vehicle2or6* v)
+    static uint8_t getZoomVolumeModifier(uint8_t zoom)
     {
-        registers regs;
-        regs.esi = X86Pointer(v);
-        call(0x0048A590, regs);
-        return { static_cast<SoundId>(regs.eax), { regs.ecx, regs.edx, regs.ebx } };
+        return std::min<uint8_t>(zoom, 2) * 35;
```

Comment:
So many magic numbers here, should we put them all into an `AudioConstants` file/class?

Reply:
I think most would be specific to this file so could just have them at the top of the file